src/main/java/frc/robot/RobotContainer.java

```diff
@@ -99,6 +98,22 @@ private void controllerBindings() {
       } else {
         tankSubsystem.setTankDrivePowers(-joystickLeft.getY(), -joystickRight.getY());
       }
+
+      // "A" button controls open and closing of the claw
+      if (mechControlXbox.getAButtonReleased()) {

```

Comment:
.get_ButtonReleased() is a method that returns whether the button has been released since the last call, and as such needs to be called iteratively. This method controllerBindings() is only called once, and is used for binding listeners.. we are able to use it for tank because we only have to bind the default command once, and the command based system will call the default command (by default)

Reply:
we'd only use this method if there was like an .onPressed(<our function here>), where we're setting up a listener instead of checking for something every iteration. lmk if that doesn't make sense

Reply:
edited code; used .whenPressed and a lambda function for A/B/X controller bindings, as described in [documentation](https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html)